frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -852,14 +854,40 @@ export const createOrUpdateResources = async (
   return responses;
 };
 
+export const filterDeployedResources = (resources: K8sResourceKind[]) =>
+  resources.filter(
+    (resource) =>
+      resource.kind === DeploymentModel.kind ||
+      resource.kind === DeploymentConfigModel.kind ||
+      (resource.kind === knSvcModel.kind &&
+        resource.apiVersion === `${knSvcModel.apiGroup}/${knSvcModel.apiVersion}`),
```

Comment:
Need to update the code accordingly as you removed the `knSvcModel` import.
```suggestion
      (resource.kind === KnServiceModel.kind &&
        resource.apiVersion === `${KnServiceModel.apiGroup}/${KnServiceModel.apiVersion}`),
```